### ISSUE TITLE
type on page to directly find matches in the page

### ISSUE THREAD

**Issue by michelecafagna26** [brave/brave-browser#1943]
## Description 
like Mozzilla firefox, find something  on the page just typing what you want to search

**Comment by srirambv** [comment]
Do you mean when you load the page and just start typing it should trigger search functionality and highlight the words?

**Comment by michelecafagna26** [comment]
Yes, something like "Find while typing on a page". It's shown more clearly here : https://support.mozilla.org/en-US/kb/search-contents-current-page-text-or-links?redirectlocale=en-US&redirectslug=Searching+within+a+page#w_find-while-typing-on-a-page

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
